test/features/showandhide/basicFields.feature

```diff
@@ -0,0 +1,39 @@
+@functional @showandhide
+Feature: Hide and show of basic fields
+
+  Background:
+    Given I have logged in
+
+  Scenario Outline: fields on the same page visible when show & hide condition is met based on mandatory and optional fields
+    Given a case type containing conditionals exists
+    And I start createCase event
+    When I enter '<value>' into the text field with id '<fieldId>'
```

Comment:
should we have  a step that verifies the fields are initially hidden?

Reply:
Done